### DISCUSSION TITLE
Clips running in Slow-Motion?!

### DISCUSSION THREAD

**Discussion by Minglarn** [blakeblackshear/frigate#1843] [Ask A Question] [answered]
Hi... Witch RC3 my clips suddenly started to go in Slow-motion and are extremely choppy. 
I'm seeing this phenomena on my ReoLink RLC510A camera only, and it only happens when going from night-mode to daylight mode. It's like the frame-rate is increasing in the camera... 
Using the recomended settings from Blake for the Reolink cams. 
Any one else seeing this?

![Half_speed_1 0](https://user-images.githubusercontent.com/58695742/134761305-0cee8e96-6668-4142-949a-5e969d9d462d.gif)

**Comment by Minglarn** [comment, marked as answer]
A restart and changed settings in ffmpeg args solved the issue... All fine.

**Reply by bm55** [reply]
what is your ffmpeg args? I am having similar issue with another Reolink
